tests/test_project/tests/test_admin.py

```diff
@@ -125,3 +127,40 @@ def test_receive_url_field_in_change(self):
         ma = ProjectAdmin(Project, self.site)
         url = ma.receive_url(p)
         self.assertContains(response, url)
+
+    def test_customize_admin_theme_css(self):
+        url = reverse('admin:index')
+
+        # test for improper configuration : not a list
+        with self.assertRaises(ImproperlyConfigured):
+            setattr(settings, 'OPENWISP_ADMIN_THEME_CSS', "string instead of list")
```

Comment:
be consistent with the quoting style, the file is using mostly single quotes, so use single quotes.